samples/highcharts/chartchooser/continuous-distribution-map/demo.html

```diff
@@ -0,0 +1,10 @@
+<div id="container" style="height: 720px; width: 800px; margin: 0 auto; text-align:center; line-height: 520px">
+    Downloading map...
+  </div>
+  
+  <script src="https://code.highcharts.com/maps/highmaps.js"></script>
+  <script src="https://code.highcharts.com/maps/modules/data.js"></script>
+  <script src="https://code.highcharts.com/maps/modules/data.js"></script>
+  <script src="https://code.highcharts.com/maps/modules/exporting.js"></script>
+  <script src="https://code.highcharts.com/maps/modules/offline-exporting.js"></script>
+  <script src="https://code.highcharts.com/mapdata/countries/us/us-all-all.js"></script>
```

Comment:
Use TopoJSON map. See the updated us-counties demo.

Reply:
Done 👍